### PR TITLE
[AC-9513] Add Sitetree integration

### PR DESCRIPTION
**Pull request by kenware**
How to test:

- On the accelerate folder
- Run make migrate
- Check any startup profile pages, you should see the top nav items

Siblings PR: https://github.com/masschallenge/accelerate/pull/3311

### CODE REVIEW

**Review by iankigen — COMMENTED** [review]
From QA
- A migration exists to add “Progress” top nav bar for the startup experience, as seen in the [mockup](https://www.figma.com/file/bs17rxAKhA4gLTq6Xq10by/Profiles?node-id=13%3A1014) ✅ 
- The startup update edit ✅  and detail ❌  views have appropriate top navigation

**Comment by kenware** [comment]
> From QA
> 
> * A migration exists to add “Progress” top nav bar for the startup experience, as seen in the [mockup](https://www.figma.com/file/bs17rxAKhA4gLTq6Xq10by/Profiles?node-id=13%3A1014) ✅
> * The startup update edit ✅  and detail ❌  views have appropriate top navigation

Checkout the sibling PR https://github.com/masschallenge/accelerate/pull/3311 @iankigen

**Review by iankigen — COMMENTED** [review]
QA looks good

File: accelerator/migrations/0106_migrate_startup_top_nav.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        alias='startup_profile_manager_subnav').first()
+    if not nav_tree:
+        nav_tree = NavTree.objects.create(
+            title='Startup Profile Manager',
+            alias='startup_profile_manager_subnav')
+
+    for item in nav_items:
+        item['tree_id'] = nav_tree.id
+        nav_item = NavTreeItem.objects.filter(
+            alias=item['alias'],
```
Comment (AmosWels) [inline accelerator/migrations/0106_migrate_startup_top_nav.py:44]: could you add `url=item['url']`, to enforce uniquenness

File: accelerator/migrations/0106_migrate_startup_top_nav.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        {"title": 'Progress',
+         'alias': 'progress',
+         'urlaspattern': True,
+         'url': 'startup_update_view startup.id'},
+        {"title": 'Team',
+         'alias': 'startup_team',
+         'urlaspattern': True,
+         'url': 'startup_team_view startup.id'},
+    ]
+    nav_tree = NavTree.objects.filter(
```
Comment (jonkiparsky) [inline accelerator/migrations/0106_migrate_startup_top_nav.py:34]: Not a blocker, but we'd typically use a `get_or_create` here and below

**Review by AmosWels — APPROVED** [review]
changes look good to me

**Review by iankigen — APPROVED** [review]

**Pull request merged**